src/Equinor.ProCoSys.IPO.Command/InvitationCommands/InvitationHelper.cs

```diff
@@ -15,6 +17,11 @@ public static string GetBaseUrl(string pcsBaseUrl, string plant)
 
         public static bool ParticipantIsSigningParticipant(ParticipantsForCommand participant) 
             => participant.Organization != Organization.External && participant.Organization != Organization.Supplier;
+        public static async Task<bool> HasIpoAdminPrivilege(IPermissionCache permissionCache, string plant, Guid currentUser)
```

Comment:
I think it may be better to have currentUserProvider and IPermissionCache in constructor here instead of passing them in as arguments

Reply:
same with PlantProvider